python/lsst/obs/base/ingest_tests.py

```diff
@@ -154,3 +159,17 @@ def testFailOnConflict(self):
         self.runIngest()
         with self.assertRaises(Exception):
             self.runIngest()
+
+    def testWriteCuratedCalibrations(self):
+        """Test that we can ingest the curated calibrations"""
+        if self.curatedCalibrationDatasetTypes is None:
+            raise unittest.SkipTest("Class requests disabling of writeCuratedCalibrations test")
+
+        self.instrument.writeCuratedCalibrations(self.butler)
+
+        dataId = {"instrument": self.instrument.getName()}
+        for datasetTypeName in self.curatedCalibrationDatasetTypes:
+            with self.subTest(dtype=datasetTypeName, dataId=dataId):
+                datasets = list(self.butler.registry.queryDatasets(datasetTypeName, collections=...,
+                                                                   dataId=dataId))
+                self.assertGreater(len(datasets), 0, f"Checking {datasetTypeName}")
```

Comment:
I'm a bit reticent to have the number be an exact match since I don't want the test to fail every time a calibration scientist adds a new file to a data package. Even having an "at least this many" number there risks breakage if someone decides that there should be fewer calibration files in the data package.